### PR TITLE
w3m: add page

### PR DESCRIPTION
**Pull request by rcorrie**
http://w3m.sourceforge.net/

### CODE REVIEW

File: pages/common/w3m.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# w3m
+
+> A text based Web browser and pager.
```
Comment (waldyrious) [inline pages/common/w3m.md]: I believe "text-based" is the correct form. Also, the "...and pager" is probably more puzzling than helpful to beginners. Is it really needed?

**Comment by waldyrious** [comment]
That's a nice start, thanks :) please check out the inline comment.

File: pages/common/w3m.md (hunk 1 above)
Comment (rcorrie) [inline pages/common/w3m.md]: @waldyrious agreed on both counts. Will update shortly.

**Comment by waldyrious** [comment]
Looks good 👍

**Pull request merged**